src/types/mod.rs

```diff
@@ -587,6 +587,16 @@ impl JsObject {
             Handle::new_internal(JsObject(local))
         }
     }
+
+    #[cfg(feature = "napi-8")]
```

Comment:
Documentation would be helpful. If that's not something you are comfortable writing, I can help.